fix the url structure and make it compatible with old urls (#43)
wireapp/wire-docs: src/how-to/associate/deeplink.rst

      "title" : "Production"
   }

> There is no requirement for these hosts to be consistent, e.g. the REST endpoint could be `wireapp.pineapple.com` and the team setting `teams.banana.com`. (But of course if you follow the rest of this documentation it most likely will be consistent)

Reviewer: ```suggestion
There is no requirement for these hosts to be consistent, e.g. the REST endpoint could be `wireapp.pineapple.com` and the team setting `teams.banana.com`. If you have been following this documentation closely, these hosts will likely be consistent in naming, regardless.
```
